### DISCUSSION TITLE
JK BMS on/off button

### DISCUSSION THREAD

**Discussion by mr-manuel** [Louisvdw/dbus-serialbattery#442] [Q&A] [answered]
Hello,

I saw some JK BMS that were delivered with a button to turn it on and off. The button is connected to the LCD port of the BMS. Unfortunately my JK BMS was not shipped with such a button. Maybe someone can tell me which pins I have to short to turn on/off the BMS?

![grafik](https://user-images.githubusercontent.com/12592765/215408437-209fdcba-8828-4166-9fef-f1072d6332bd.png)

**Comment by mr-manuel** [comment, marked as answer]
Now I tested a bit and found out, that you have to shorten the LCD pins `K-` and `K+` to power on the BMS. To turn it off shorten the same pins for about 5 seconds.

**Comment by vincegod** [comment]
Do you know what the connector is so I can order one?

**Reply by mr-manuel** [reply]
Unfortunately not
